src/index.js

```diff
@@ -124,7 +124,7 @@ class CompressionPlugin {
             const sub = {
               file,
               path: parse.pathname,
-              query: parse.query || '',
+              query: parse.query ? `?${parse.query}` : '',
```

Comment:
Why is this change needed ? Bugfix ?

Reply:
Bug fix https://github.com/webpack-contrib/compression-webpack-plugin/issues/105

Reply:
Always better to do this in a separate PR for `CHANGELOG` generation reasons and to keep things clear

Reply:
@michael-ciniawsky it is break many tests and it is very inconvenient in this case, also it is not difficult write CHANGELOG manually in this cases